tests/acceptance/administrator/_steps/Joomla3Steps/StockRoomManagerJoomla3Steps.php

```diff
@@ -17,112 +17,113 @@
  */
 class StockRoomManagerJoomla3Steps extends AdminManagerJoomla3Steps
 {
-    /**
-     * Function to add a new Stock Room Detail
-     *
-     * @param   string $name Name of the Stock Room
-     * @param   string $minAmount Min Amount
-     *
-     * @return void
-     */
-    public function addStockRoom($name = 'Sample', $minAmount = '100')
-    {
-        $I = $this;
-        $I->amOnPage(\StockRoomManagerJoomla3Page::$URL);
-        $I->click(\StockRoomManagerJoomla3Page::$newButton);
-        $I->waitForElement(\StockRoomManagerJoomla3Page::$stockRoomName, 30);
-        $I->fillField(\StockRoomManagerJoomla3Page::$stockRoomName, $name);
-        $I->fillField(\StockRoomManagerJoomla3Page::$minimumStockAmount, $minAmount);
-        $I->waitForElement(\StockRoomManagerJoomla3Page::$saveButton, 30);
-        $I->click(\StockRoomManagerJoomla3Page::$saveButton);
-        $I->waitForText(\StockRoomManagerJoomla3Page::$stockRoomSuccessMessage, 60, \StockRoomManagerJoomla3Page::$selectorSuccess);
-        $I->click(\StockRoomManagerJoomla3Page::$closeButton);
-    }
+	/**
+	 * Function to add a new Stock Room Detail
+	 *
+	 * @param   string $name Name of the Stock Room
+	 * @param   string $minAmount Min Amount
+	 *
+	 * @return void
+	 */
+	public function addStockRoom($name = 'Sample', $minAmount = '100')
+	{
+		$I = $this;
+		$I->amOnPage(\StockRoomManagerJoomla3Page::$URL);
```

Comment:
@NguyenTrang091296, pls help add use `StockRoomManagerJoomla3Page` and remove `\` and make all for the rest. Thanks!